packages/flutter/lib/src/widgets/media_query.dart

```diff
@@ -109,6 +110,15 @@ class MediaQueryData {
   ///  * [MediaQuery.textScaleFactorOf], a convenience method which returns the
   ///    textScaleFactor defined for a [BuildContext].
   final double textScaleFactor;
+  
+  /// The current brightness mode of the host platform.
+  /// 
+  /// For example, starting in Android Pie, battery saver mode asks all apps to
+  /// render in a "dark mode". That change is reflected in this property.
```

Comment:
This seems to just get you the current value? What do you mean by "change" here?

Reply:
I deleted the statement about "change"